app/code/core/Mage/Sales/Model/Resource/Quote.php

```diff
@@ -156,12 +156,14 @@ public function getReservedOrderId($quote)
     /**
      * Check is order increment id use in sales/order table
      *
-     * @param string $orderIncrementId
+     * @param string|integer $orderIncrementId
+     *
      * @return boolean
      */
     public function isOrderIncrementIdUsed($orderIncrementId)
     {
         $adapter   = $this->_getReadAdapter();
+        $orderIncrementId = $adapter->prepareColumnValue($fields['increment_id'], $orderIncrementId);
```

Comment:
Where does `$fields` come from?

Reply:
Oh crap, sorry about that.. Forgot one line